Makefile

```diff
@@ -1,5 +1,6 @@
 OUT ?= ./build
 SRC := src
+SRC += -I remotedcurl
```

Comment:
Is it necessary to create a new directory for remote interface? I expect single or two files added.

Reply:
I expect three files, remotedcurl.c -> remote-worker.c, remote-common.c and remote-interface.c. Yes, It's not necessary.